.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - name: Set up Go 1.16
       uses: actions/setup-go@v2
       with:
-        go-version: ^1.16
```

Comment:
didn't notice this before. it was loosely marked as ^1.14 which actually ran as 1.16 which is where this issue surfaced:  https://github.com/golang/go/issues/44129